app/routes/wallet/jws.py

```diff
@@ -23,15 +23,36 @@
     "/sign",
     response_model=JWSCreateResponse,
     summary="Sign JWS",
-    description="""
-Sign JSON Web Signature (JWS)
-
-See https://www.rfc-editor.org/rfc/rfc7515.html for the JWS spec.""",
 )
 async def sign_jws(
     body: JWSCreateRequest,
     auth: AcaPyAuth = Depends(acapy_auth_from_header),
 ) -> JWSCreateResponse:
+    """
+    Sign JSON Web Signature (JWS)
+    ---
+
+    The difference between the did and verification_method fields is
+    that if the `did` field is used, acapy will make an educated guess
+    about which key associated with the did to use to sign the jwt.
+
+    While with the `verification_method` field, the user is explicitly
+    specifying which key to use to sign the jwt.
+
+    See https://www.rfc-editor.org/rfc/rfc7515.html for the JWS spec.
+
+    Request body:
+    ---
+        JWSCreateRequest: The JWS to sign.
+            did: str: The DID to sign the JWS with.
+            header: dict: The header of the JWS.
+            payload: dict: The payload of the JWS.
+            verification_method: str: The verification (did with key to use) method to use.
```

Comment:
JWSCreateRequest: 
-   did/verification_method are optional
-    header**s** also optional
-    payload has a default {} in our models, but we should probably remove that considering the payload is required and I guess signing {} won't make sense
- the descriptions should be a bit more informative. We repeat JWS over and over. "The payload of the JWS". Or, is it the payload being signed?
- lastly, I think this model can be a bit confusing, giving that one of did/verification method can be specified, or _both_, so we should probably include an example of this last case so people can see what types of values are passed